stdlib/public/core/Codable.swift

```diff
@@ -5518,6 +5518,16 @@ internal struct _DictionaryCodingKey: CodingKey {
     self.stringValue = "\(intValue)"
     self.intValue = intValue
   }
+
+  fileprivate init(codingKey: CodingKey) {
+    self.stringValue = codingKey.stringValue
+    self.intValue = codingKey.intValue
+  }
+}
+
+public protocol CodingKeyRepresentable {
```

Comment:
```suggestion
@available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
public protocol CodingKeyRepresentable {
```

Reply:
Thanks - I was uncertain about this 'future release' annotation!

Reply:
> I was uncertain about this 'future release' annotation!

The placeholder version number is documented in:

<https://github.com/apple/swift/blob/main/docs/StandardLibraryProgrammersManual.md#availability>